doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -206,10 +206,13 @@ In order for the storage solution(s) to be conformant with the Reference Archite
 
 |Ref|Specification|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.stg.001`||||
-|`ra2.stg.002`||||
-|`ra2.stg.003`||||
-|`ra2.stg.004`||||
+|`ra2.stg.001`| Ephemeral Storage | An implementation must support ephemeral storage, for the unpacked container images to be stored and executed from, as a directory in the filesystem on the worker node on which the container is running <br>See the [Container runtimes](#4.4) section above for more information how this meets the requirement for ephemeral storage for containers <br>Ephemeral storage exists for the life of a container<br>Kubernetes Volumes attached using a Volume plugin can also be used to deliver Ephemeral Storage (may require additional tasks to ensure dleetion of ephemral storge on teh shutting down of a container) ||
```

Comment:
```suggestion
|`ra2.stg.001`| Ephemeral Storage | An implementation must support ephemeral storage, for the unpacked container images to be stored and executed from, as a directory in the filesystem on the worker node on which the container is running <br>See the [Container runtimes](#4.4) section above for more information how this meets the requirement for ephemeral storage for containers <br>Ephemeral storage exists for the life of a container<br>Kubernetes Volumes attached using a Volume plugin can also be used to deliver Ephemeral Storage (may require additional tasks to ensure deletion of ephemeral storage on the shutting down of a container) ||
```

Reply:
@CsatariGergely Sorry, I don't see a suggested change. 

"on" is missing from "... section above for more information how this meets the requirement for ephemeral storage ..." ---> "... section above for more information on how this meets the requirement for ephemeral storage ..."